pkg/build/controller/strategy/custom_test.go

```diff
@@ -66,7 +66,7 @@ func TestCustomCreateBuildPod(t *testing.T) {
 	errorCases := map[int][]string{
 		0: {"BUILD", string(buildJSON)},
 	}
-	standardEnv := []string{"SOURCE_URI", "SOURCE_REF", "OUTPUT_IMAGE", "OUTPUT_REGISTRY"}
+	standardEnv := []string{"SOURCE_URI", "SOURCE_REPOSITORY", "SOURCE_CONTEXT_DIR", "SOURCE_REF", "OUTPUT_IMAGE", "OUTPUT_REGISTRY"}
```

Comment:
SOURCE_URI - https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/util.go#L54
IIRC, initially we set it for users to get the repo information.